Tests/PSGetFindModule.Tests.ps1

```diff
@@ -55,6 +56,134 @@ function SuiteCleanup {
     $null = Import-PackageProvider -Name PowerShellGet -Force
 }
 
+Describe FindModuleTests -Tags "TDD" {
+    
+    BeforeAll {
+        SuiteSetup
+    }
+
+    AfterAll {
+        SuiteCleanup
+    }
+
+    
+    # Purpose: Validate Find-Module (on a module with stable and prerelease versions)
+    #
+    # Action: Find-Module
+    #
+    # Expected Result: Find-Module should return the latest stable version, not the latest prerelease version of the module
+    #
+    It FindModuleReturnsLatestStableVersion {
+        $psgetModuleInfo = Find-Module -Name $script:PrereleaseTestModule -Repository Local
```

Comment:
>-Repository Local [](start = 74, length = 17)

It looks like you are testing against you local repo.
Is it possible to let me know whether I should review this test file now?